server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilities.java

```diff
@@ -132,6 +159,17 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (nonAggregatableIndices != null) {
             builder.field(NON_AGGREGATABLE_INDICES_FIELD.getPreferredName(), nonAggregatableIndices);
         }
+        if (meta.isEmpty() == false) {
+            builder.startObject("meta");
+            for (Map.Entry<String, Set<Object>> entry : meta.entrySet()) {
```

Comment:
what about predictable order for the keys?  Looks like `meta` was built as HashMap in `FieldCapabilities.Builder`